exercises/colorsquare.js

```diff
@@ -53,21 +64,31 @@ class ColorSquare{
 			yes, this will cause a cascade of all the elements to the right
 			make sure the rightNeighbor is something!  the rightmost element won't have a neighbor
 		*/
-	handleClick(){
-
+	handleClick(event){
+        this.colorIndex++;
+        if (this.colorIndex > this.colorsArray.length-1){
```

Comment:
you could also say this.colorIndex  === this.colorsArray.length since that would be the last one